### DISCUSSION TITLE
Removing Enterprise Linux components

### DISCUSSION THREAD

**Discussion by sbluhm** [uyuni-project/uyuni#8861] [Devel] [answered]
What are you thoughts on the technical debts of the Enterprise Linux efforts?

Last build was still installable on a Enterprise Linux OS. But now with the container version, the code not required anymore.

I would slowly start removing the implemented Enterprise Linux code to reduce the technical debts, allow cleaner future developments and reduce potential regressions.

**Comment by agraul** [comment, marked as answer]
To me it sounds good to remove the code if it's not needed for the containerized version. Less code is always better :smile:

**Comment by rjmateus** [comment]
up vote on removing the code :+1:

**Comment by sbluhm** [comment]
thank you for your thoughts. will start working on it
